acc_test/import_resource_test.go

```diff
@@ -257,8 +257,8 @@ func TestKafkaUserResource_importBasic(t *testing.T) {
 			},
 			{
 				PreConfig: func() {
-					fmt.Println("Sleep for 6 minutes to wait for Kafka cluster to be ready for resize.")
-					time.Sleep(6 * time.Minute)
+					fmt.Println("Sleep for 10 minutes to wait for Kafka cluster to be ready for resize.")
```

Comment:
We have to be careful of this, the tests have a hard stop at the 2 hour mark (if my memory serves correctly) at which point they fail.  Not telling you to remove, just letting you know.

Reply:
1 hour mark actually :) 
Since we've removed a whole bunch of tests that revolved around dependencies, we should have freed up a lot of time in our tests so we might be able to afford adding 4 minutes here. Ideally though we would be doing a polled check with a timeout limit rather than a hard wait like this. On further thought I think I'd rather comment this test out than have to add to the wait.